fpgaconvnet/models/network/Network.py

```diff
@@ -200,3 +201,27 @@ def load_network(self, network_path):
         # update partitions
         self.update_partitions()
 
+    def check_network_validity(self):
```

Comment:
nice! I think the name of the function might be a bit too broad or vague, as validity might refer to many things. How about something like "check_graph_completeness"

Reply:
Yeah, that is fair enough. I will rename this one to "check_graph_completeness" as you suggested.